# Settings of PCL

PCLDistanceThreshold: 0.1
PCLMaxNumPlane: 8
PCLMinInlier: 15


# Settings of RHT

AccumulatorChoice: 1 # 1: array, 2: ball
AccumulatorRhoResolution: 0.1
AccumulatorThetaResolution: 0.1
AccumulatorPsiResolution: 0.1
AccumulatorThreshold: 0
AccumulatorKMaxSuppress: 5 # set to zero to deactivate

RHTMaxIter: 1000000
RHTTolDist: 3    # makes sure points are not too far apart
RHTMinArea: 0.5
RHTPlaneNumber: 8 # number of principal planes, set to zero if all planes should be shown


#Settings of iterative RHT

iterAccumulatorChoice: 2 # 1: array, 2: ball
iterAccumulatorRhoResolution: 0.05
iterAccumulatorThetaResolution: 0.02
iterAccumulatorPsiResolution: 0.02
iterAccumulatorKMaxSuppress: 5 # set to zero to deactivate
iterAccumulatorMinThreshold: 0

iterRHTIter: 1000000
iterRHTTolDist: 3   # makes sure points are not too far apart
iterRHTMinArea: 0.5
iterRHTPlaneNumber: 8 # number of principal planes, set to zero if all planes should be shown
iterRHTNumPlaneperIter: 1


# Settings of CGAL Region Growing
CGALRegionGrowMinNumInliers: 80
CGALRegionGrowMaxDistToPlane: 0.07
CGALRegionGrowMaxDistBetwPoint: 10
CGALRegionGrowMaxDiffNormalThreshold: 0.9

CGALRGRegulActive: true
CGALRGRegulParall: true
CGALRGRegulOrthog: true
CGALRGRegulCoplanar: false
CGALRGRegulParallOrthTol: 10
CGALRGRegulCoplanarTol: 0.2



